addons/binding/org.openhab.binding.tankerkoenig/META-INF/MANIFEST.MF

```diff
@@ -10,6 +10,7 @@ Import-Package:
  com.google.common.collect,
  com.google.gson,
  org.apache.commons.io,
+ org.apache.commons.lang;version="2.6.0",
```

Comment:
Is there a reason to use this specific version?

Reply:
No! That was created automatically  and I missed to remove it.